images/terraform/ecr.tf

```diff
@@ -49,19 +49,6 @@ module "ecr_nginx_frontend_identity" {
   }
 }
 
-module "ecr_nginx_grafana" {
-  source = "./repo_pair"
-
-  namespace = local.namespace
-  repo_name = "nginx_grafana"
-
-  description = "An nginx image for reverse proxying Grafana"
-
-  providers = {
-    aws.ecr_public = aws.ecr_public
-  }
-}
-
```

Comment:
I'm not really sure what this was for but I didn't find any references to it and I suspect it wasn't used, so hopefully it's safe to remove.

Reply:
This creates both a public and private ECR repo. I believe it is safe to remove.